### PR TITLE
Include the Catalyst vs ASF roles distinction in the Catalyst Program

### PR DESCRIPTION
**Pull request by michaelsembwever**
```
 patch by Mick Semb Wever; reviewed by Dinesh Joshi, Paulo Motta
```

Lots of trailing whitespace removal here.  See comment for main change.

### CODE REVIEW

File: site-content/source/modules/ROOT/pages/cassandra-catalyst-program.adoc [hunk 1]
```diff
@@ -119,6 +119,17 @@ Catalysts will be recognized for their ongoing work to foster a more collaborati
...
+----
+[discrete]
+=== ASF Compliance
+[discrete]
+==== Catalyst vs. ASF Committer and PMC roles
+
+The Catalyst award is not to be confused with the project’s committership or being on its PMC, roles that involve project participation and contributions. The Catalyst program is recognition for effort of any type around the project, and is not a position or title within the project or the ASF.
+
+----
+
```
Comment (michaelsembwever) [inline site-content/source/modules/ROOT/pages/cassandra-catalyst-program.adoc:132]: main change.

**Comment by michaelsembwever** [comment]
@dineshjoshi @pauloricardomg , waiting on review from you both.

**Review by pauloricardomg — APPROVED** [review]

**Comment by pauloricardomg** [comment]
👍 lgtm

**Review by dineshjoshi — APPROVED** [review]
+1, thanks!

**Pull request merged**